(GH-8618) Add note to Measure-Object in 7.3 (#8638)
MicrosoftDocs/PowerShell-Docs: reference/7.3/Microsoft.PowerShell.Utility/Measure-Object.md

## NOTES

> Starting in PowerShell 7.3, `Measure-Object` does not return an error when
> processing an object which is missing the property being measured except in
> **StrictMode**. When `Measure-Object` processes an object which is missing the
> specified property in **StrictMode** it returns a
> `System.Management.Automation.PSArgumentException` error for the **Property**
> argument.

Reviewer: ```suggestion
Starting in PowerShell 7.3, `Measure-Object` no longer returns an error when
processing an object that is missing the property being measured unless you are
running in **StrictMode**. In **StrictMode**, `Measure-Object` returns a
`System.Management.Automation.PSArgumentException` when processing an object
that is missing the specified property.
```
